src/System.Text.RegularExpressions/ref/System.Text.RegularExpressions.csproj

```diff
@@ -4,6 +4,7 @@
   <PropertyGroup>
     <OutputType>Library</OutputType>
     <NuGetTargetMoniker>.NETStandard,Version=v1.7</NuGetTargetMoniker>
+    <DefineConstants Condition="'$(TargetGroup)' == 'netcoreapp1.1'">$(DefineConstants);netcoreapp11</DefineConstants>
   </PropertyGroup>
```

Comment:
Yes please, the other additions are less lines of code so #if statements are fine. That's not the case here.